Fix failing static.SmokeTest.test_landing test

## Overview

The tests are currently failing on the ``master`` branch.

## Expected Behavior

We should endeavour to keep the testsuite passing at all times.

## Technical details

```
$ ./manage.py test
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
..F...
======================================================================
FAIL: test_landing (backend.static.tests.SmokeTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lamby/git/work/drivendata/challenge-application-test/interface/backend/static/tests.py", line 9, in test_landing
    self.assertContains(resp, 'Hello')
  File "/home/lamby/git/work/drivendata/challenge-application-test/interface/.venv/lib/python2.7/site-packages/django/test/testcases.py", line 393, in assertContains
    self.assertTrue(real_count != 0, msg_prefix + "Couldn't find %s in response" % text_repr)
AssertionError: Couldn't find 'Hello' in response

----------------------------------------------------------------------
Ran 6 tests in 0.049s

FAILED (failures=1)
Destroying test database for alias 'default'...
```

## Acceptance criteria

The testsuite passes.

--- a/interface/backend/static/tests.py
+++ b/interface/backend/static/tests.py
@@ -6,4 +6,5 @@ class SmokeTest(TestCase):
     def test_landing(self):
         url = reverse('static:home')
         resp = self.client.get(url)
-        self.assertContains(resp, 'Hello')
+        self.assertContains(resp, 'Concept to Clinic')
+        self.assertEqual(resp.status_code, 200)

